### ISSUE TITLE
[Bug] Conditional field appears on Anki but not on AnkiDroid when it is called "Type"

### ISSUE THREAD

**Issue by linguax-gc** [ankidroid/Anki-Android#7991]
###### Reproduction Steps

1. Add this deck (1 note) [test script.zip](https://github.com/ankidroid/Anki-Android/files/5758157/test.script.zip)
2. Open in Anki and check both conditional fields are showing on the back of the card (first should say "Yes" and second "B")
3. Sync with AnkiDroid
4. Open in AnkiDroid.
5. Card doesn't show conditional field called "Type"
6. Rename field to anything else in Anki
7. Sync
8. Open in AnkiDroid
9. Now the conditional works

###### Expected Result
AnkiDroid should show the the result whatever the name of the field is, I guess. 

###### Actual Result
Only shows if name is not "Type".

###### Debug info
AnkiDroid Version = 2.14.2

Android Version = 10

ACRA UUID = 6c37bf56-6819-42ea-aa3d-59c19c372897

Scheduler = std2

Crash Reports Enabled = true

###### Research
- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))
(Only tested in Beta)
I was going crazy until I realized what was happening!

**Comment by Arthur-Milchior** [comment]
Hi,

Thanks for the report.
Anki changed its way of processing card a few months ago. We'll eventually copy it, but it's a long process. Given that "Type" is a specific keyword in anki and that renaming "Type" to "Typ" or "type" is easy, I doubt will correct it soon, sorry.

**Comment by linguax-gc** [comment]
Sure, no problem, thanks for the quick reply. I changed the field name and everything was fine, fortunately, so it's not a a big issue. Just reporting it in case it wasn't registered somewhere and in case someone else found this problem and didn't realize the issue was the field name as I did. I didn't see it mentioned anywhere so I just kept trying to review the card template in fear I mistyped something. :)
